gen/calc.py

```diff
@@ -421,7 +425,10 @@ def validate_os_type(os_type):
         },
         'rexray_config_method': {
             'file': {
-                'must': {'rexray_config_contents': calculate_rexray_config_contents},
+                'must': {'rexray_config_contents': calculate_rexray_config_from_file},
+            },
+            'string': {
+                'must': {'rexray_config_contents': calculate_rexray_config_from_string},
             },
```

Comment:
The impact to upgrades would be if we removed this option, users would have to replace their `rexray_config_method: aws` with `file` or `string` and provide their REX-Ray config explicitly. "That sounds fine to me" == keep (and add a test for) the `aws` option?